### ISSUE TITLE
make the count down for IdleRPGs vote clock visible.

### ISSUE THREAD

**Issue by liquidspeaker** [DiscordBotList/issues#207]
#### Suggestion

I, and several others think you need to make the obviously tied to each voter individually. You should make the countdown clock for idleRPG and, any other time released  vote you service, a visible clock.

- people will be able to get their vote in at the same two decent times a day instead of over guessing and ending up voting the firs time at 9:40am and then again, due to simply being told to come back in 12 hours every time they come back to check, voting again at 11:30pm
- Improve all users experience, who reliably vote, having visual count down, and will help stay on a schedule that doesn't press back by 5 minutes to an hour everytime they vote. This leaves the most dedicated to the official bot server getting less and less sleep until they have to simply let  a loot crate go, which is not really fair to them.

TLDR:A visible count down clock would just make everyone who uses the voting systems life a lit easier and more beneficial.

**Comment by tonkku107** [comment]
Duplicate of #104
